Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift

```diff
@@ -65,15 +66,20 @@ class ConfirmYourBidPINViewController: UIViewController {
                         .updateUserCredentials(loggedInProvider)
                         .mapReplace(provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { provider -> Observable<BypassResults> in
```

Comment:
This is the big addition, but it doesn't _look_ like an addition. That's because we were calling `updateUserCredentials` twice for no reason, and I removed it in https://github.com/artsy/eidolon/commit/a096ff10cff5ee6e300598051381fac5f521c20c

Logic for making the network call and determining if CC bypass should be made is contained completely within the network model. Returning an Observable of a tuple is a new thing I'm trying, not sure if I like it or not.